.github/workflows/flink-ci.yml

```diff
@@ -70,8 +70,9 @@ jobs:
   flink-scala-2-12-tests:
     runs-on: ubuntu-22.04
     strategy:
+      fail-fast: false
       matrix:
-        jvm: [8, 11]
+        jvm: [8, 11, 17]
```

Comment:
Flink only added Java 17 since 1.18. so the matrix here can be complicated.
https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/java_compatibility/#java-17

Reply:
added exclude for jvm 17 and flink 1.17